### ISSUE TITLE
BehaviorSpace: allow arbitrarily computed parameter spaces

### ISSUE THREAD

**Issue by SethTisue** [NetLogo/NetLogo#1477]
over at #1475 @arthurhjorth wrote:

> I'm even finding myself using BehaviorSpace less, because it's so hard to do specific subsets of the parameter space, but relatively easy to write in code. But that's a story for another time.

here's a ticket for your story, Arthur :-)

**Comment by arthurhjorth** [comment]
Yes please! This is turning out to be a pretty great day already!

**Comment by nicolaspayette** [comment]
There is also a bit of discussion about this over at https://github.com/NetLogo/NetLogo/issues/576#issuecomment-211820050.

**Comment by ERSUCC** [comment]
This has been implemented since 6.4, and is known as the sub experiment syntax. See [this](https://ccl.northwestern.edu/netlogo/docs/behaviorspace.html#subexperiment-syntax) section of the BehaviorSpace docs for details.
